odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -614,12 +614,22 @@ def get_available_options(self):
 
     @api.model
     def value_to_html(self, value, options=None):
+        if not value:
+            return ''
         barcode_symbology = options.get('symbology', 'Code128')
         barcode = self.env['ir.actions.report'].barcode(
             barcode_symbology,
             value,
             **{key: value for key, value in options.items() if key in ['width', 'height', 'humanreadable']})
-        return u'<img src="data:png;base64,%s">' % base64.b64encode(barcode).decode('ascii')
+
+        img_element = html.Element('img')
+        for k, v in options.items():
+            if k.startswith('img_'):
+                img_element.set(k[4:],v)
```

Comment:
how come pylint doesn't complain? :thinking: 

```suggestion
                img_element.set(k[4:], v)
```

Reply:
And why he did not complain locally is a mystery too.

Reply:
> how come pylint doesn't complain? 🤔

Ho, I know, because this isn't master and we took the wonderfull decision to skip pylint on stables (prepare to cry when reaching master)